### ISSUE TITLE
scylla_setup: Cannot get driver information: No such device

### ISSUE THREAD

**Issue by tzach** [scylladb/scylla#3658]
*Installation details*
Scylla version (or git commit hash): 2.2
Cluster size: 1
OS (RHEL/CentOS/Ubuntu/AWS AMI): CentOS on AWS, i3.4xlarge

scylla_setup prompt for *Amazon EC2 configuration check* and return with a `Cannot get driver information: No such device` error

```
$ sudo scylla_setup
Skip any of the following steps by answering 'no'
Do you want to run Amazon EC2 configuration check?
Answer yes to have this script verify that current instance configuration is optimized for running Scylla; answer no to skip this check.
[YES/no]
Cannot get driver information: No such device
cat: /sys/class/net/eth0/address: No such file or directory
VPC is not enabled!
To enable enhanced networking, please enable VPC.
```

**Comment by tzach** [comment]
@syuu1228 can we close this issue?

**Comment by syuu1228** [comment]
@tzach Yes, https://github.com/scylladb/scylla/commit/6fca92ac3c407e1a9e42ca870246e9fdb8fc2fbc should fix the issue.

**Comment by powellchristoph** [comment]
I ran into this today with version `2.2.0-0.20180705.240b9f122-0ubuntu1~xenial`.

It says this was fixed and backported into `2.2`. Has a new build been released?

**Comment by tzach** [comment]
@powellchristoph it will be part of the next 2.2.x release, 2.2.1
Hopefully in a few weeks
